tables:
  table0:
    num_persons: 0
    status: "unvisited"
    order: []
    location: 
      position: {x: -0.74, y: -3.57, z: 0.0}
      orientation: {x: 0.0, y: 0.0, z: -0.99, w: 0.03}
      semantic: "end"
    objects_cuboid:
      min: [-3.30, -4.0, 0.0]
      max: [-1.4, -3.0, 8.0]
    persons_cuboid:
      min: [-3.80, -4.50, 0.0]
      max: [-0.9, -2.50, 8.0]
current_table: "table0"
